### PR TITLE
Update the spatial awareness docs

### PR DESCRIPTION
**Pull request by wiwei**
- Add an explicit guide on how to disable/hide (feedback from some folks around looking for keywords like 'hide' don't give any hits)
- Make it clearer that the steps on the page aren't necessary for users of the default profile, and also provide a note about the default HL2 profile

### CODE REVIEW

File: Documentation/SpatialAwareness/SpatialAwarenessGettingStarted.md [hunk 1]
```diff
@@ -82,11 +83,14 @@ This feature enables rapid development and testing without requiring the build a
 to do final acceptance testing using a built and deployed version of the application, running on the target 
 hardware and platform.
 
+### Hiding the spatial mesh
+
+In order to hide the spatial mesh (assuming the default profile is in use, or the steps in [Enable Spatial Awareness](#enable-spatial-awareness))
```
Comment (keveleigh) [inline Documentation/SpatialAwareness/SpatialAwarenessGettingStarted.md]: Too many )s

image

```suggestion
In order to hide the spatial mesh (assuming the default profile is in use, or the steps in [Enable Spatial Awareness](#enable-spatial-awareness)
```

File: Documentation/SpatialAwareness/SpatialAwarenessGettingStarted.md [hunk 2]
```diff
@@ -82,11 +83,14 @@ This feature enables rapid development and testing without requiring the build a
 to do final acceptance testing using a built and deployed version of the application, running on the target 
 hardware and platform.
 
+### Hiding the spatial mesh
+
+In order to hide the spatial mesh (assuming the default profile is in use, or the steps in [Enable Spatial Awareness](#enable-spatial-awareness)
```
Comment (davidkline-ms) [inline Documentation/SpatialAwareness/SpatialAwarenessGettingStarted.md]: This is not quite accurate. Hiding the mesh is different from disabling the system.  Either the caption needs to be Disabling Spatial Awareness or the instructions need to direct the user to set the DisplayOption on the observer to None

Comment (wiwei) [reply Documentation/SpatialAwareness/SpatialAwarenessGettingStarted.md]: Removed this, added a code snippet to the usage guide.

**Comment by wiwei** [comment]
@davidkline-ms can you please take another look?

**Pull request merged**